.github/actions/run-publish-vr-screenshot/action.yml

```diff
@@ -84,58 +84,64 @@ runs:
         path: screenshots
 
     # ==========================================================
-    # STEPS BELOW WILL FAIL TO RUN ON GITHUB ACTIONS - see TODOs
+    # STEPS BELOW WILL FAIL TO RUN ON GITHUB ACTIONS - see @TODOs
```

Comment:
commenting steps that would not work with this composite action for future reference for us and VRT team